frontend/src/common/queries/filter.ts

```diff
@@ -324,11 +330,12 @@ function aggregateCollectionDatasetRows(
       ...new Set(aggregatedCategoryValues.development_stage_ancestors),
     ],
     disease: uniqueOntologies(aggregatedCategoryValues.disease),
+    organism: uniqueOntologies(aggregatedCategoryValues.organism),
```

Comment:
Corrected `ESLint: Expected object keys to be in ascending order. 'organism' should be before 'self_reported_ethnicity'.(sort-keys)` linting error.

Reply:
Not sure if you use VSCode, if so, I use this plugin to sort the keys!

https://marketplace.visualstudio.com/items?itemName=zengxingxin.sort-js-object-keys